src/cache/cache_las.c

```diff
@@ -330,6 +352,51 @@ __wt_las_cursor_close(
 	return (ret);
 }
 
+/*
+ * __las_remove_block --
+ *	Remove all records for a given page from the lookaside store.
+ */
+static int
+__las_remove_block(WT_SESSION_IMPL *session,
+    WT_CURSOR *cursor, uint32_t btree_id, uint64_t pageid, uint64_t *decrp)
+{
+	WT_CONNECTION_IMPL *conn;
+	WT_DECL_RET;
+	WT_ITEM las_key;
+	uint64_t las_counter, las_pageid;
+	uint32_t las_id;
+
+	*decrp = 0;
+
+	conn = S2C(session);
+
+	__wt_writelock(session, &conn->cache->las_sweepwalk_lock);
+
+	/*
+	 * Search for the block's unique prefix and step through all matching
```

Comment:
The term 'unique prefix' brings other things to mind. How about "the block's unique page and btree identifiers"? Or even just "the block's unique identifiers".?

Reply:
I went with:
```
Search for the block's unique btree ID and page ID prefix and step
through all matching records, removing them.
```